### DISCUSSION TITLE
2017/01/30/harveys-counterfeit-beacon

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#5] [Announcements]
# 2017/01/30/harveys-counterfeit-beacon

http://localhost:4000/2017/01/30/harveys-counterfeit-beacon

**Comment by jengalas** [comment]
Posted by Terry Smith
March 8, 2022 at 6:19 am
_Imported from WordPress_

I am a private citizen trying to transfer a 1939 tower (very similar to yours)and beacon from a local paper company site to the town of Ticonderoga airport. In order for the town to accept this gift they need to know the weight of this tower. Could you help me with this? It is a 55’ tower with a 24” rotating beacon. Anything would help. Thank you.

**Comment by jengalas** [comment]
Posted by Frank Ward
December 16, 2021 at 3:31 pm
_Imported from WordPress_

“Old Airmen never die, they just fly away.”
Served 2874th GEEIA Squadron, 60-63, Ramstein AB, Germany
what a ride…
A1C Frank Ward
Baytown, Texas

**Comment by jengalas** [comment]
Posted by Robin R Ehrhardt
September 14, 2021 at 2:59 pm
_Imported from WordPress_

My spouse and i are part owners of the Beaver Marsh airport near Chemult Oregon and are in the second year of a lighting project and water source for EMS and Firefighting.
Might I ask where to find those VERY EXPENSIVE lens for the beacon or what your recall of the cost was?

**Comment by jengalas** [comment]
Posted by Travis T.
February 25, 2020 at 11:58 am
_Imported from WordPress_

Howdy!

Wow! This is such a fantastic project, and amazing restoration. Someday I’d like to do a setup similar to yours. As a lighting and aviation history enthusiast, it’s great to see others who have similar interests and hobbies. Did you custom build some kind of flasher unit for the obstruction light above the beacon? I also see you found a good spot to mount your AcuRite station!

**Comment by jengalas** [comment]
Posted by Tim wilson
November 24, 2019 at 2:31 pm
_Imported from WordPress_

I am looking for 2 light bulbs and something that looks like a coil on an older car, only taller. Would like a diagram on how to wire it. Here is my information that I got off the beacon. BEacon DEA 022, ID FAA L801, 120 V WATTS 1750 MAN.P/N 0200, APPOLLO LIGHTINGS CO. Columbus,Ohio . THanks for any help!

**Comment by jengalas** [comment]
Posted by George Wilson
September 2, 2017 at 4:22 pm
_Imported from WordPress_

Hi,
Do you know where I might find information on an antique Crouse Hindes search light? I l can send a photo if you write to my e-mail address.
Thanks!
gw

**Comment by jengalas** [comment]
Posted by Christian Ackerley
February 23, 2017 at 2:24 am
_Imported from WordPress_

Harvey –

Thanks for a great article. This is really inspiring!

I own an early 500 watt Westinghouse “on course” light. As you probably know, these early navigation aides were mounted below the rotating beacons on the early air mail navigation towers.

Steve at the Cibola County Museum in Gallup, New Mexico was able to establish, from the serial number of my Westinghouse “on course” light, that it was manufactured in 1930. My “on course” light came from the small town of Blair, located near Omaha, Nebraska.

My light is complete and original, except for the interior lens, which is missing. This is a 10″ lens. I have been told these interior lenses came in red, green, and clear colorations.

Do you have any idea where I could obtain a 10″ interior lens for my Westinghouse “on course” light? I don’t need the mirror, I just need the interior lens. I will settle for any color I can find.

I will appreciate any information you can provide.

Thank You!

Christian Ackerley
Phoenix, Arizona

**Reply by jengalas** [reply]
Posted by Jason lewis
June 11, 2017 at 1:15 am
_Imported from WordPress_

Christian, my name is Jason and I am in Florence Arizona, I have a restored course hinds beacon with a code disk and I am in search of both a red and green course light. Let me know if you are interested in selling.

**Comment by jengalas** [comment]
Posted by Harvey Hartman
January 31, 2017 at 6:36 pm
_Imported from WordPress_

Man, that Harvey is one super-talented dude! His wife should let him spend more money on worthwhile projects like this!!!
